client/pages/Login.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {useState, useEffect} from 'react';
+import { KeyboardAvoidingView, StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import { useTheme } from '@react-navigation/native';
+import { useNavigation } from '@react-navigation/core';
+
+import { firebase, auth, provider, signInWithGoogle } from '../configs/firebase';
+
+const styles = StyleSheet.create({
```

Comment:
I haven't been able to figure out if adding style sheet elements at the top or bottom of the file is the appropriate convention...thoughts? Might also be a separate file and import thing, I'll look into it more.

Reply:
I'm indifferent about top or bottom, but I like the idea of maybe importing from a separate file, if we happen to have reusable styles. Not sure what's best practice.

Reply:
I think at the bottom is standard.